### PR TITLE
save configuration

### PR DESCRIPTION
**Pull request by mauroLonghi**

### CODE REVIEW

File: mobile/src/main/java/com/yalantis/watchface/manager/ConfigurationManager.java [hunk 1]
```diff
@@ -120,15 +123,24 @@ private void updateConfigurationFile() throws IOException {
...
         if (!mediaStorageDir.exists()) {
             mediaStorageDir.mkdirs();
         }
-        File mediaFile = new File(mediaStorageDir.getPath() + File.separator + fileName + ".png");
+        File mediaFile = new File(mediaStorageDir.getPath() + File.separator + fileName + PATH_EXTENSION);
         return mediaFile;
     }
+
+    public int getConfigItem(String key) {
+        int configItem = 0;
```
Comment (nicolbidone) [inline mobile/src/main/java/com/yalantis/watchface/manager/ConfigurationManager.java]: D:

File: mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java [hunk 2]
```diff
@@ -92,6 +99,7 @@ public void sendNotification(ConfigurationActivity configurationActivity) {
         notificationBuilder.setAutoCancel(true);
```
Comment (nicolbidone) [inline mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java:99]: D:

File: mobile/src/main/java/com/yalantis/watchface/task/SendToDataLayerThread.java [hunk 3]
```diff
@@ -30,13 +31,13 @@ public void run() {
         NodeApi.GetConnectedNodesResult nodes = Wearable.NodeApi.getConnectedNodes(mGoogleApiClient).await();
         ByteArrayOutputStream stream = new ByteArrayOutputStream();
         bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
```
Comment (nicolbidone) [inline mobile/src/main/java/com/yalantis/watchface/task/SendToDataLayerThread.java]: D:

**Review by nicolbidone — CHANGES_REQUESTED** [review]

File: mobile/src/main/java/com/yalantis/watchface/manager/ConfigurationManager.java (hunk 1 above)
Comment (mauroLonghi) [reply mobile/src/main/java/com/yalantis/watchface/manager/ConfigurationManager.java]: Done

File: mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java (hunk 2 above)
Comment (mauroLonghi) [reply mobile/src/main/java/com/yalantis/watchface/presenter/configuration/ConfigurationPresenterImpl.java:99]: Done

File: mobile/src/main/java/com/yalantis/watchface/task/SendToDataLayerThread.java (hunk 3 above)
Comment (mauroLonghi) [reply mobile/src/main/java/com/yalantis/watchface/task/SendToDataLayerThread.java]: Done

**Review by nicolbidone — APPROVED** [review]

**Pull request merged**